src/core/Akka/Dispatch/Dispatchers.cs

```diff
@@ -57,7 +57,13 @@ internal sealed class FullThreadPoolExecutorServiceImpl : ThreadPoolExecutorServ
         /// <param name="run">TBD</param>
         public override void Execute(IRunnable run)
         {
+#if NETSTANDARD
             ThreadPool.UnsafeQueueUserWorkItem(Executor, run);
+#else
+            // use native .NET 6 APIs here to reduce allocations
+            // preferLocal to help reduce context switching
+            ThreadPool.UnsafeQueueUserWorkItem(run, true);
```

Comment:
Major change here - leverages new overload to the `ThreadPool.UnsafeQueueUserWorkItem` method added in .NET Core 3.0: https://docs.microsoft.com/en-us/dotnet/api/system.threading.threadpool.unsafequeueuserworkitem?view=net-6.0